spring-cloud-etcd-discovery/src/main/java/org/springframework/cloud/etcd/discovery/EtcdDiscoveryClient.java

```diff
@@ -57,28 +66,44 @@ public ServiceInstance getLocalServiceInstance() {
 	}
 
 	@Override
-	@SneakyThrows
```

Comment:
This needs to turn into a rethrow a runtime exception. Use `ReflectionUtils.rethrowRuntimeException`.